packaging/linux/common/notes.appdata.xml.in

```diff
@@ -0,0 +1,98 @@
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>MPL-2.0</project_license>
+    <name>@PROJECT_NAME@</name>
+    <developer_name>Ruby Mamistvalove</developer_name>
+    <update_contact>ruby.mamistvalove_AT_gmail.com</update_contact>
+    <summary>Note-taking application, write down your thoughts</summary>
+    <url type="homepage">https://www.get-notes.com/</url>
+    <url type="bugtracker">https://github.com/nuttyartist/notes/issues</url>
+    <url type="donation">https://github.com/sponsors/nuttyartist</url>
+    <description>
+        <p>Notes is an open source and cross-platform note-taking app that is both beautiful and powerful.</p>
+        <p>Features overview:</p>
+        <ul>
+            <li>Native app (written in C++ with Qt)</li>
+            <li>Fast with a low memory footprint</li>
+            <li>Beautiful and sleek looking, yet still powerful</li>
+            <li>Fully open source and cross-platform (Linux, macOS, Windows)</li>
+            <li>Completely private: Tracks nothing</li>
+            <li>Folders and tags: Organize your ideas hierarchically using nested folders and universally using tags</li>
+            <li>Markdown Support: Format text without lifting your hands from the keyboard</li>
+            <li>Different themes: Switch between Light, Dark, and Sepia</li>
+            <li>Feed View: Select multiple notes to see them all one after another in the editor</li>
+            <li>Always runs in the background: Use the hotkey <code>Super</code> + <code>Shift</code> + <code>N</code> to summon Notes</li>
+        </ul>
+    </description>
+    <screenshots>
+        <!-- TODO: Upload these screenshots to a better place. -->
+        <screenshot type="default">
+            <image>https://user-images.githubusercontent.com/626206/222927528-64941f78-4093-4d3d-8000-fd0dca61980a.png</image>
+            <caption>Light theme</caption>
+        </screenshot>
+        <screenshot>
+            <image>https://user-images.githubusercontent.com/626206/222927531-a52cd062-acd1-49b2-9b58-543ff5772c6e.png</image>
+            <caption>Editor settings</caption>
+        </screenshot>
+        <screenshot>
+            <image>https://user-images.githubusercontent.com/626206/222927543-6c91792f-3fa3-430b-94b7-c2ff92f6ed4d.png</image>
+            <caption>Dark theme</caption>
+        </screenshot>
+        <screenshot>
+            <image>https://user-images.githubusercontent.com/626206/222927546-1d333c84-2d82-4e2d-8c96-a15f42d56af2.png</image>
+            <caption>Sepia theme</caption>
+        </screenshot>
+        <screenshot>
+            <image>https://user-images.githubusercontent.com/626206/222930768-87a1f5ef-1707-45c3-9322-fb0e64869cc2.png</image>
+            <caption>Compact mode</caption>
+        </screenshot>
+    </screenshots>
+    <content_rating type="oars-1.1" />
+    <releases>
+        <release date="2023-03-05" version="2.1.0">
```

Comment:
Note to self: I'm assuming `2.1.0` gets released today, but if it doesn't, make sure to update the `date` property here.

Reply:
If everything goes well, then tomorrow (: